vzstorage-pd/vendor/github.com/virtuozzo/goploop-cli/ploopvol/ploop-volume.go

```diff
@@ -0,0 +1,65 @@
+package ploopvol
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+)
+
+var (
+	ploopVolumeBinary = "/usr/sbin/ploop-volume"
+)
+
+type PloopVolume struct {
+	Path string
```

Comment:
Who has to set Path? Can we have something like func Open(volumePath string) (PloopVolume, error)?